kuberos.go

```diff
@@ -86,22 +86,9 @@ type StateFn func(*http.Request) string
 func defaultStateFn(secret []byte) StateFn {
 	// Writing to a hash never returns an error.
 	// nolint: errcheck, gas
-	return func(r *http.Request) string {
```

Comment:
I think you removed this line by mistake. The code won't compile with it removed.

Reply:
That's right, thanks for catching this.